tests/tools/nnpackage_run/src/randomgen.cc

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "randomgen.h"
+#include "nnfw.h"
+#include "nnfw_util.h"
+#include "misc/RandomGenerator.h"
+
+#include <iostream>
+
+namespace nnpkg_run
+{
+
+template <class T> void randomData(nnfw::misc::RandomGenerator &randgen, void *data, uint64_t size)
+{
+  for (uint64_t i = 0; i < size; i++)
+    reinterpret_cast<T *>(data)[i] = randgen.generate<T>();
+}
+
+void RandomGenerator::generate(std::vector<Allocation> &inputs)
+{
+  // generate random data
+  const int seed = 1;
```

Comment:
How about changing `seed` not static but dynamic? I mean such as time()...

Reply:
@YongseopKim Thank you for comment.

First, it is beyond this PR.

Second, there has been discussion about constant seed vs. changing seed. I am for const since it gives deterministic result. If we use different seed for each CI run, it is hard to know the regression comes from different seed or PR itself.

If someone or we would like to strengthen the test by testing more input values (it is good), we may add additional argument like #2622, and run several times with different seed daily between midnight to dawn when CI is likely to be idle.